packages/libs/lambda-at-edge/types.d.ts

```diff
@@ -11,6 +11,20 @@ export type DynamicPageKeyValue = {
   };
 };
 
+// Image optimization
+export type ImageConfig = {
+  deviceSizes: number[];
+  imageSizes: number[];
+  loader: "default" | "imgix" | "cloudinary" | "akamai";
```

Comment:
Do we need `imgix`, `cloudinary`, `akamai` here?

Reply:
Actually I didn't try these other loaders, will check on it but I think they go directly to those URLs